src/plugins/date/date.c

```diff
@@ -465,6 +465,7 @@ int elektraDateGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEKTRA_UN
 			       keyNew ("system/elektra/modules/date/exports", KEY_END),
 			       keyNew ("system/elektra/modules/date/exports/get", KEY_FUNC, elektraDateGet, KEY_END),
 			       keyNew ("system/elektra/modules/date/exports/set", KEY_FUNC, elektraDateSet, KEY_END),
+			       keyNew ("system/elektra/modules/date/export/validateKey", KEY_FUNC, validateKey, KEY_END),
```

Comment:
Its a Elektra key, we should avoid camelcase here. So either `validate` (should be enough?) or `validate/key` (if we might need further validation methods but seems unlikely: validating of keyset is already `kdbSet`). And there is a typo: `export` -> `exports`.